pkg/api/domain.go

```diff
@@ -3,11 +3,16 @@ package api
 import (
 	"fmt"
 	"strings"
+
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 const (
-	ClusterNameAPICI = "api.ci"
-	ClusterNameAPPCI = "app.ci"
+	ClusterNameAPICI   = "api.ci"
+	ClusterNameAPPCI   = "app.ci"
+	ClusterNameBuild01 = "build01"
+	ClusterNameBuild02 = "build02"
+	ClusterNameVSphere = "vsphere"
```

Comment:
Cluster names seem to be unrelated to "domains", why include them in this file?

Reply:
Moved them into `constant.go` in the same pkg.